tests/unit/testplan/report/test_testing.py

```diff
@@ -341,6 +383,21 @@ def test_report_json_serialization(dummy_test_plan_report):
     check_report(actual=deserialized_report, expected=dummy_test_plan_report)
 
 
+def test_report_json_binary_serialization(
+        dummy_test_plan_report_with_binary_asserts
+    ):
+    """JSON Serialized & deserialized reports should be equal."""
+    test_plan_schema = TestReportSchema(strict=True)
+    data = test_plan_schema.dumps(
+        dummy_test_plan_report_with_binary_asserts
+    ).data
```

Comment:
It would be nice to check the contents of the serialized data - make sure that the data is encoded as base64 correctly.